packages/playground/src/components/pull-request/sections/pull-request-merge-section.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from 'react'
+import { Text } from '@harnessio/canary'
+import { isEmpty } from 'lodash-es'
+import {
+  NavArrowUp,
+  NavArrowDown,
+  WarningTriangleSolid,
+  CheckCircleSolid,
+  Clock,
+  ChatBubble
+} from '@harnessio/icons-noir'
+import { CheckStatus, TypeCheckData } from '../interfaces'
+import { timeDistance } from '../utils'
+
+interface PullRequestMergeSectionProps {
+  checkData: TypeCheckData[]
+  checksInfo: { header: string; content: string; status: CheckStatus }
+}
+const PullRequestMergeSection = ({ checkData, checksInfo }: PullRequestMergeSectionProps) => {
+  const [isExpanded, setExpanded] = useState(false)
+
+  return (
+    !isEmpty(checkData) && (
+      <div className=" pt-4 pb-3  border-b">
```

Comment:
Same, this might cause reformatting issues due to extra spacing.

Reply:
do u have an extension that formats when that happens? @vardanbansal-harness

Reply:
I think this needs to be done manually, I don't know of any extensions that can do it.

Reply:
okay fixed